### ISSUE TITLE
关于weibo数据集预处理的一些问题

### ISSUE THREAD

**Issue by Qi2yU** [yutchina/MMDFND#14]
您好，我对于您网盘分享的预处理过的weibo数据集有一些疑惑：
1.我发现您关于weibo数据预处理的代码并未处理文本，只处理了图片，但是从网盘上下载下来的预处理过的数据集的新闻文本也发生了一些改变，比如文本中的标点符号基本上都没了，这是否会影响实验结果呢？
2.网盘上预处理过的数据集包含post_id,image_id,content,post_text,label,event_label,category字段，请问post_text字段有什么作用呢？我看感觉是把content新闻文本分解成了一些词组
3. 预处理过的数据集的post_id字段是由科学计数法表示的，与原数据对不上，这个字段应该不重要吧~
4. 预处理过的数据集中的图片train_loader.pkl和新闻train_origin.csv如何对应呢？我看了下train_loader.pkl只包含处理过的图片，而没有索引，那它如何与新闻对应上呢？因为train_origin.csv一条新闻有的有多张图片。是按train_origin.csv中的新闻顺序排列的吗~

谢谢！

**Comment by yutchina** [comment]
你好。关于第一个问题，文本的标点符号是根据baseline做的数据清洗。如果你需要完整的文本，你可以去weibo的github找找。关于第二个和第三个问题，确实存在一些没有用到的字段，这些字段不重要。关于第四个问题，数据的预处理是因为读取完整的图片数据集并且转换为clip的嵌入等等需要非常非常久的时间，所以我提前把这些数据通过代码进行预处理，这样在加载图片数据的时候就不需要再处理一遍了。模型在运行的时候会读取文本文件和预处理好的图片文件，所以你不需要担忧顺序的问题。

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2025年04月24日 16:27 |
| 收件人 | yutchina/MMDFND ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [yutchina/MMDFND] 关于weibo数据集预处理的一些问题 (Issue #14) |
Qi2yU created an issue (yutchina/MMDFND#14)

您好，我对于您网盘分享的预处理过的weibo数据集有一些疑惑：
1.我发现您关于weibo数据预处理的代码并未处理文本，只处理了图片，但是从网盘上下载下来的预处理过的数据集的新闻文本也发生了一些改变，比如文本中的标点符号基本上都没了，这是否会影响实验结果呢？
2.网盘上预处理过的数据集包含post_id,image_id,content,post_text,label,event_label,category字段，请问post_text字段有什么作用呢？我看感觉是把content新闻文本分解成了一些词组
3. 预处理过的数据集的post_id字段是由科学计数法表示的，与原数据对不上，这个字段应该不重要吧~
4. 预处理过的数据集中的图片train_loader.pkl和新闻train_origin.csv如何对应呢？我看了下train_loader.pkl只包含处理过的图片，而没有索引，那它如何与新闻对应上呢？因为train_origin.csv一条新闻有的有多张图片。是按train_origin.csv中的新闻顺序排列的吗~

谢谢！

**Comment by Chenwanyu-desinger** [comment]
你好，老师 ，我们在做的时候还需要与处理吗
